BEIMA.Backend/DeviceFunctions/AddDevice.cs

```diff
@@ -34,6 +35,14 @@ public static class AddDevice
             log.LogInformation("C# HTTP trigger function processed a device post request.");
 
 
+            var authService = AuthenticationDefinition.AuthticationInstance;
+            var claims = authService.ParseToken(req);
+
+            if(claims == null)
```

Comment:
```suggestion
            if(AuthenticationDefinition.AuthenticationInstance.ParseToken(req) == null)
```
I think it would be ideal if we could condense this as much as possible, since we are going to have to put this on every endpoint.

Reply:
nvm, I see that we have to use claims.Username for the LastModified field below.